.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make build
       - run: make test-integration-cover
-      - run: /go/bin/goveralls -coverprofile=coverage-all.out -service=circle-ci -repotoken=$COVERALLS_TOKEN
+      - run: /go/bin/goveralls -coverprofile=coverage-all.out -service=circle-ci -repotoken=647ITQUZn7dM7Jvcu8NUWYMAkMgqKWkaO
```

Comment:
The token should not be public, otherwise, everyone with this token could send wrong data to coveralls.io, it's not a big security issue but it's better to keep it private in an Environment Variable

Reply:
I agree. However, it was not working so I added here as it is other projects.

Reply:
I think this is the reason why you don't see the environment variable:
![image](https://user-images.githubusercontent.com/7964685/63427293-1825d280-c415-11e9-988e-81631e6cf26f.png)
because otherwise, it's present in the settings:
![image](https://user-images.githubusercontent.com/7964685/63427354-3be91880-c415-11e9-81fd-d81bb8335fa3.png)

Reply:
Yep. Done tks.